x/bep3/types/keys.go

```diff
@@ -22,16 +22,20 @@ const (
 	DefaultParamspace = ModuleName
 )
 
+// DefaultLongtermStorageDuration is 1 week (assuming a block time of 7 seconds)
+const DefaultLongtermStorageDuration int64 = 86400
+
 // Key prefixes
 var (
-	AtomicSwapKeyPrefix     = []byte{0x00} // prefix for keys that store AtomicSwaps
-	AtomicSwapByBlockPrefix = []byte{0x01} // prefix for keys of the AtomicSwapsByBlock index
-	AssetSupplyKeyPrefix    = []byte{0x02} // prefix for keys that store global asset supply counts
+	AtomicSwapKeyPrefix             = []byte{0x00} // prefix for keys that store AtomicSwaps
+	AtomicSwapByBlockPrefix         = []byte{0x01} // prefix for keys of the AtomicSwapsByBlock index
+	AssetSupplyKeyPrefix            = []byte{0x02} // prefix for keys that store global asset supply counts
+	AtomicSwapLongtermStoragePrefix = []byte{0x04} // prefix for keys of the AtomicSwapLongtermStorage index
```

Comment:
```suggestion
	AtomicSwapLongtermStoragePrefix = []byte{0x03} // prefix for keys of the AtomicSwapLongtermStorage index
```
nit: use sequential key prefixes

Reply:
https://github.com/Kava-Labs/kava/pull/397/commits/6eea7377ae5b59fa2b7a13b6acd998cd71d16c75